runtime/doc/Makefile

```diff
@@ -151,88 +151,91 @@ os_risc.txt:
 os_win32.txt:
 	touch $@
 
-# Note that $< works with GNU make while $> works for BSD make.
-# Is there a solution that works for both??
+# In *BSD, the variable ‘$<’ is used in suffix-transformation rules (in GNU this
```

Comment:
Can you please use normal single apostrophes? Not `U+2018 LEFT SINGLE QUOTATION MARK`?

Reply:
Fixed it. I'm sorry, please.